astropy/units/format/vounit.py

```diff
@@ -126,6 +125,16 @@ def _parse_unit(cls, unit, detailed_exception=True):
 
         return cls._units[unit]
 
+    @classmethod
+    def _validate_unit(
+        cls,
+        unit: str,
+        detailed_exception: bool = True,
+        class_name: str | None = "VOUnit",
+    ) -> None:
+        if unit not in cls._custom_units:
```

Comment:
Ah, now I see how this works. That also means the super with the class name is fairly natural here.